src/main/kotlin/io/easybreezy/project/application/issue/command/language/Normalizer.kt

```diff
@@ -1,33 +1,35 @@
 package io.easybreezy.project.application.issue.command.language
 
 import com.google.inject.Inject
-import io.easybreezy.project.application.issue.command.language.element.ElementNormalizer
+import io.easybreezy.project.application.issue.command.language.normalizer.ElementNormalizer
 import io.easybreezy.project.model.issue.Label
 import io.easybreezy.project.model.issue.Priority
 import java.util.UUID
 
 class Normalizer @Inject constructor(
     private val elementNormalizers: Set<@JvmSuppressWildcards ElementNormalizer>
 ) {
-    suspend fun normalize(parsedIssue: ParsedIssue, project: UUID): NormalizedIssue {
-
-        var normalizedIssue = NormalizedIssue(
-            parsedIssue.title,
-            parsedIssue.description
-        )
+    suspend fun normalize(parsed: ParsedFields, project: UUID, fields: List<String>): NormalizedFields {
```

Comment:
"fields" param add complexity to understanding how normalizer works. Will be better to call each normalizer what you need independent. Like use Labels normalizer as constructor parameter or  like Normalizer.get<LabelNoramilzer>().normalize(...)